Storage/Processor/Processor.php

```diff
@@ -61,6 +68,19 @@ public function getResponse(Request $request, string $hash, string $configHash,
         return $response;
     }
 
+    /**
+     * @param array<string, mixed>  $configArray
+     */
+    public function configFactory(string $hash, array $configArray): Config
+    {
+        $normalizedArray = ArrayTool::normalizeAndSerializeArray($configArray);
+        if ($normalizedArray === false) {
```

Comment:
idealy your normalizedArray throws an error instead of returning false. 
You can catch that error and
1. customize it in this case
2. decide to continue with your "else path" if throwing an error is not necessary (in other locations)